src/libs/portfolio/interfaces.ts

```diff
@@ -32,6 +32,13 @@ export type TokenResult = {
   }
 }
 
+export type GasTankTokenResult = TokenResult & {
+  availableAmount: bigint
+  cashback: bigint
+  saved: bigint
+  isFirstCashbackConfettiVisible: boolean
```

Comment:
If we go with https://github.com/AmbireTech/ambire-common/pull/1135/files#r1939732041, I'd alter this to `firstCashbackSeenAt: timestamp | null`